server/app/controllers/dev/seeding/DatabaseSeedTask.java

```diff
@@ -46,11 +46,11 @@ public final class DatabaseSeedTask {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseSeedTask.class);
   private static final int MAX_RETRIES = 10;
-  private static final ImmutableList<QuestionDefinition> CANONICAL_QUESTIONS =
+  private static final ImmutableList<QuestionDefinition> SAMPLE_QUESTIONS =
       ImmutableList.of(
           new QuestionDefinitionBuilder()
               .setQuestionType(QuestionType.NAME)
-              .setName("Name")
+              .setName("Sample Name Question")
```

Comment:
Yes, it looks like it might, in ways that are untested. Some of the well known paths are only used in tests and some in source code. I'll revert these question name changes and leave this for a future PR.